.github/workflows/internal-archive-release.yml

```diff
@@ -139,31 +207,50 @@ jobs:
         env:
           SECRETS_CONTEXT: ${{ toJson(secrets) }}
 
-      - name: "Install Spark Dependencies"
-        if: ${{ contains(github.repository, 'dbt-labs/dbt-spark') }}
-        run: |
-          sudo apt-get update
-          sudo apt-get install libsasl2-dev
 
       - name: "Set up Python - ${{ env.PYTHON_TARGET_VERSION }}"
         uses: actions/setup-python@v5
         with:
           python-version: ${{ env.PYTHON_TARGET_VERSION }}
 
-      - name: "Install python tools"
+      - name: "Install Python Dependencies"
         run: |
           python -m pip install --user --upgrade pip
           python -m pip --version
           python -m pip install tox
           tox --version
 
-      - name: "Run Tests"
+      - name: "Run Tests using tox"
         run: tox
 
+
+  run-integration-tests-hatch:
+    name: 'Integration Tests (Hatch)'
+    needs: [initial-setup, job-setup]
+    if: "${{ needs.initial-setup.outputs.is_hatch_adapter == 'true' }}"
+
+    uses: "dbt-labs/dbt-postgres/.github/workflows/integration-tests.yml@main"
+    with:
+      core_branch: "main"
+      dbt_adapters_branch: "main"
```

Comment:
Yes since this is more about getting around a problem with how postgres has dependencies with core and dbt-adapters. In the future we might have this an overrideable field but for now it is not a blocker since the circumstances where this comes up in the versionless world is very very rare.